packages/dev/core/src/Rendering/GaussianSplatting/gaussianSplatting.ts

```diff
@@ -376,4 +416,17 @@ export class GaussianSplatting {
         this.mesh?.dispose();
         this.mesh = null;
     }
+
+    private getTextureSize(length: number): Vector2 {
+        let dim = 2;

```

Comment:
You should not have a `width` > maximum supported texture width (`engine.getCaps().maxTextureSize`).

I think the best way to compute the texture dimensions is to set `width=engine.getCaps().maxTextureSize` and compute `height` accordingly.